.github/workflows/npm-publish.yml

```diff
@@ -8,11 +8,25 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Login to NPM
-        run: npm login ${{ secrets.npm_token }}
-      - name: Publish package
-        run: |
-          cd packages/tauri-js
-          echo "We will publish this directory."
-          ls # npm publish
+    - uses: actions/checkout@v2
+    - name: Use Node 12
+      uses: actions/setup-node@v1
+      with:
+        # specify node version and the registry for the RELEASE build
+        node-version: 12
+        registry-url: https://registry.npmjs.org/
+    - name: Build package
+      run: |
+        npm install -g yarn
+        yarn install
+        yarn rollup -c
```

Comment:
are we using rollup?

Reply:
that is a good point. i am using it at tauri-forage - and think we should migrate to it from webpack - but at the moment it shouldn't be here until we do that. in fact, this all needs to be revised. Marking as WIP